lib/telemetry_ui.ex

```diff
@@ -102,7 +102,12 @@ defmodule TelemetryUI do
   def theme(name), do: config(name, :theme)
   def backend(name), do: config(name, :backend)
 
+  def valid_share_key?(share_key) do
+    is_binary(share_key) and byte_size(share_key) === 16
```

Comment:
Is `16` used elsewhere when we **generate** a share key? Should it not be duplicated here?

Reply:
I found what I was looking for:

* https://github.com/mirego/telemetry_ui/blob/master/lib/telemetry_ui/web/crypto.ex#L5
* https://github.com/mirego/telemetry_ui/blob/master/lib/telemetry_ui/web/crypto.ex#L8

So, shouldn’t all those `16` literals be stored as a single constant?